compiler-rt/lib/builtins/riscv/restore.S

```diff
@@ -79,20 +100,34 @@ __riscv_restore_3:
 __riscv_restore_2:
 __riscv_restore_1:
 __riscv_restore_0:
+#ifdef __CHERI_PURE_CAPABILITY__
+  clc      cs2, 0(csp)
```

Comment:
Those offsets are wrong?

Reply:
0 is correct, but the following 3 offsets are wrong, I fixed them.